src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -455,7 +455,8 @@ public void forbidRenewalOverrideWhenFirstRequestIsNotRecall() {
         "item is not renewable, " +
         "reached number of renewals limit," +
         "renewal date falls outside of the date ranges in the loan policy, " +
-        "items cannot be renewed when there is an active recall request"))));
+        "items cannot be renewed when there is an active recall request, " +
+        "Item not renewed: item is Declared lost"))));
```

Comment:
Why does this message appear in this test? The item should not be declared lost during this test and so the error should not appear either.

Reply:
If an item can't be overridden, the endpoint returns all possible errors which can be overridden.

Reply:
Similarly to my other question about formatting. The other messages do not have the `Item not renewed:` pre-amble.

I wonder if that pre-amble is added by the UI as part of the presentation.

Reply:
Yes, it is. Removed.